### DISCUSSION TITLE
Information request, does integration work with Xplora 6 Play

### DISCUSSION THREAD

**Discussion by smarthomefamilyverrips** [Ludy87/xplora_watch#154] [Q&A] [answered]
### What version of Integration has the issue?

@Ludy87 Sorry to put this as a "bug" but did not know how else to ask the following question. Does the integration also work with the Xplora 6 Play watch?

### System Health details

None

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).

### Describe the issue

None

### Reproduction steps

1.
2.
3.
...

### Debug logs

```text
None
```

**Comment by Ludy87** [comment]
In principle, it should work.

**Reply by smarthomefamilyverrips** [reply]
@Ludy87 I did order 2 of the X6 play watches and expect them end of the week, I will try and leave a message here if works (or not), thanks for your response 👍🏻

**Comment by smarthomefamilyverrips** [comment, marked as answer]
@Ludy87 I can confirm on version 2.7.1 the Xplora X6 Play watches also work 👍🏻
